templates/fields/_locale.twig

```diff
@@ -10,8 +10,8 @@
 {% endfor %}
 
 {% for key, option in locales %}
-    {% set current = option.slug == app.request.get('_locale') ? option : current %}
-    {% set currentkey = option.slug == app.request.get('_locale') ? key : currentkey %}
+    {% set current = option.get.slug == app.request.get('_locale') ? option.get.slug : current %}
```

Comment:
Shouldn't these be `option.get('slug')` (or `option.slug` works too) rather than option.get.slug?